src/patternfly/components/HelperText/helper-text.scss

```diff
@@ -1,36 +1,34 @@
+  --#{$helper-text}__item-text--Color: var(--pf-t--global--text--color--regular);
+  --#{$helper-text}__item-text--FontWeight: var(--pf-t--global--font--weight--body);
+  --#{$helper-text}__item-icon--m-indeterminate--Color: var(--pf-t--global--icon--color--subtle);
+  --#{$helper-text}__item-text--m-indeterminate--Color: var(--pf-t--global--text--color--subtle);
+  --#{$helper-text}__item-icon--m-warning--Color: var(--pf-t--global--icon--color--status--warning--default);
+  --#{$helper-text}__item-text--m-warning--FontWeight: var(--pf-t--global--font--weight--body--bold);
+  --#{$helper-text}__item-icon--m-success--Color: var(--pf-t--global--icon--color--status--success--default);
+  --#{$helper-text}__item-text--m-success--FontWeight: var(--pf-t--global--font--weight--body--bold);
+  --#{$helper-text}__item-icon--m-error--Color: var(--pf-t--global--icon--color--status--danger--default);
+  --#{$helper-text}__item-text--m-error--FontWeight: var(--pf-t--global--font--weight--body--bold);
 
   // dynamic
-  --#{$helper-text}--m-dynamic__item-icon--Color: var(--#{$pf-global}--Color--100);
-  --#{$helper-text}--m-dynamic__item-text--Color: var(--#{$pf-global}--Color--100);
-  --#{$helper-text}--m-dynamic--m-indeterminate__item-icon--Color: var(--#{$pf-global}--Color--200);
-  --#{$helper-text}--m-dynamic--m-indeterminate__item-text--Color: var(--#{$pf-global}--Color--200);
-  --#{$helper-text}--m-dynamic--m-warning__item-icon--Color: var(--#{$pf-global}--warning-color--100);
-  --#{$helper-text}--m-dynamic--m-warning__item-text--Color: var(--#{$pf-global}--Color--200);
-  --#{$helper-text}--m-dynamic--m-success__item-icon--Color: var(--#{$pf-global}--success-color--100);
-  --#{$helper-text}--m-dynamic--m-success__item-text--Color: var(--#{$pf-global}--Color--200);
-  --#{$helper-text}--m-dynamic--m-error__item-icon--Color: var(--#{$pf-global}--danger-color--100);
-  --#{$helper-text}--m-dynamic--m-error__item-text--Color: var(--#{$pf-global}--Color--100);
+  --#{$helper-text}--m-dynamic__item-icon--Color: var(--pf-t--global--icon--color--regular);
+  --#{$helper-text}--m-dynamic--m-indeterminate__item-icon--Color: var(--pf-t--global--icon--color--subtle);
+  --#{$helper-text}--m-dynamic--m-indeterminate__item-text--Color: var(--pf-t--global--text--color--subtle);
+  --#{$helper-text}--m-dynamic--m-warning__item-icon--Color: var(--pf-t--global--icon--color--status--warning--default);
+  --#{$helper-text}--m-dynamic--m-warning__item-text--FontWeight: var(--pf-t--global--font--weight--body--bold);
+  --#{$helper-text}--m-dynamic--m-success__item-icon--Color: var(--pf-t--global--icon--color--status--success--default);
+  --#{$helper-text}--m-dynamic--m-error__item-icon--Color: var(--pf-t--global--icon--color--status--danger--default);
 
   // icon
-  --#{$helper-text}__item-icon--MarginRight: var(--#{$pf-global}--spacer--xs);
-
+  --#{$helper-text}__item-icon--MarginRight: var(--pf-t--global--spacer--xs);
```

Comment:
@mcoker would we want to replace this with a gap on helper-itext__item as a followup instead? May not be viable if we want to consider the possibility of adding other elements that can be rendered inside a `helper-text__item`, but adding anything extra to a helper text may not be ideal